### DISCUSSION TITLE
Allow users to bookmark and rate scripts.

### DISCUSSION THREAD

**Discussion by stevebarakat** [johnlindquist/kit#552] [Ideas]
It would be nice if users could rate scripts, maybe just a simple "like" button...  and maybe an "add to favorites" button? 🌞

**Comment by johnlindquist** [comment]
Yeah for sure!

I'll use the "reaction" data from github discussions and display them in the "hot" tab. The ones with the most ❤️ reactions, I'll push to the top 😇!
